rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -65,20 +65,26 @@ build_rootfs() {
 	# This is done to maintain consistency
 	PACKAGES=$(echo $PACKAGES | sed  -e 's/ /,/g' )
 	EXTRA_PKGS=$(echo $EXTRA_PKGS | sed  -e 's/ /,/g' )
-	
+
 	# extra packages are added to packages and finally passed to debootstrap
 	if [ "${EXTRA_PKGS}" = ""  ]; then
 		echo "no extra packages"
 	else
 		PACKAGES="${PACKAGES},${EXTRA_PKGS}"
 	fi
-	
+
 	${PKG_MANAGER} --variant=minbase \
 		--arch=${ARCHITECTURE}\
 		--include="$PACKAGES" \
 		${OS_NAME} \
-		${ROOTFS_DIR} 
+		${ROOTFS_DIR}
 
 	chroot $ROOTFS_DIR ln -s /lib/systemd/systemd /usr/lib/systemd/systemd
+
+	# tmp.mount may not be installed by default in /usr/lib/systemd
+	if ! [ -f $ROOTFS_DIR/etc/systemd/system/tmp.mount ] && \
+	   ! [ -f $ROOTFS_DIR/usr/lib/systemd/system/tmp.mount ]; then
+		cp $ROOTFS_DIR/usr/share/systemd/tmp.mount $ROOTFS_DIR/etc/systemd/system/tmp.mount
```

Comment:
- Is `/etc/systemd` guaranteed to exist? Might be worth a `mkdir -p`?
- I'd put double quotes around all the paths "just in case".
- Not wrong but interesting to see the bangs outside the `test` :smile:

Reply:
About `/etc/systemd/system`: that's owned by systemd, so we should be good. If there isn't systemd, then all the rest of the PR doesn't make sense :D 
```
# dpkg -L systemd | grep "/etc/systemd/system" 
/etc/systemd/system
```